File: codes/python/Code_2_04_3.py
# 入力
N, S = map(int, input().split())

# 答えを求める
answer = 0
for i in range(1, N + 1):
	for j in range(1, N + 1):
		if i + j <= S:
			answer += 1

# 出力
print(answer)

File: codes/c/Code_2_04_3.c
#include <stdio.h>

int main() {
	// 入力
	int N, S;
	scanf("%d%d", &N, &S);
	
	// 答えを求める
	int answer = 0;
	int i, j;
	for (i = 1; i <= N; i++) {
		for (j = 1; j <= N; j++) {
			if (i + j <= S) {
				answer += 1;
			}
		}
	}
	
	// 出力
	printf("%d\n", answer);
}
